controllers/routemonitor/routemonitor_interface.go

```diff
@@ -0,0 +1,21 @@
+package routemonitor
+
+import (
+	"context"
+
+	routev1 "github.com/openshift/api/route/v1"
+	v1alpha1 "github.com/openshift/route-monitor-operator/api/v1alpha1"
+	utilreconcile "github.com/openshift/route-monitor-operator/pkg/util/reconcile"
+
+	ctrl "sigs.k8s.io/controller-runtime"
+)
+
+type RouteMonitorInterface interface {
```

Comment:
other than in the struct this interface is not used, so I think you could remove it

Reply:
yeah, will remove